### DISCUSSION TITLE
There should be more post about QWER Docs.

### DISCUSSION THREAD

**Discussion by SrGeneroso** [kwchang0831/svelte-QWER#19] [Ideas]
Hi, I know this being a template starter should not be too cluttered, but maybe a good way to help beginners woud be to create more posts explaining the main areas of QWER, this way, we'll have automatic discussions in the repo.

**Comment by kwchang0831** [comment]
Per your request, I have set up a documentation site @ https://docs-svelte-qwer.vercel.app/
It is working in progress.

**Reply by SrGeneroso** [reply]
Dude, you might have forgotten to add "Blazingly Fast" to the description of QWER, but your response to any request it's truly blazingly quick. Thanks a lot.

**Reply by kwchang0831** [reply]
Thanks so much for your support.

It's not yet Blazingly Fast. I am working hard toward it :D
